### PR TITLE
Update the mandatory RC2 test case list

### PR DESCRIPTION
**Pull request by collivier**
The test case descritions is now up-to-date with CNTT RC2 gates [1].

[1] https://build.opnfv.org/ci/view/functest-kubernetes/job/functest-kubernetes-v1.21-daily/22/

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 1]
```diff
@@ -40,52 +40,223 @@ run by the Kubernetes community (under the aegis of the CNCF).
...
-| Kubernetes.create_scale_and_delete_statefulset                     |
-| Kubernetes.list_namespaces                                         |
+Then CNTT RC2 starts with the test list defined by
+[K8s Conformance](https://github.com/cncf/k8s-conformance) which is expected to
+grow according to the ongoing requirement traceability.
 
-The following software versions are considered to verify Kubernetes v1.21
-(latest stable release) selected by CNTT:
+[End-to-End Testing](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md)
+basically respectively asks focus and fix regexes to select or to blacklist
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:48]: not sure what is intended here -- suggest delete "respectively"
what is meant by "asks focus and fix regexes"?

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 2]
```diff
@@ -40,52 +40,223 @@ run by the Kubernetes community (under the aegis of the CNCF).
...
+single tests:
+  - focus basically matches
+    Conformance or [Testing Special Interest Groups](https://github.com/kubernetes/community/blob/master/sig-testing/charter.md)
+    in the following tabs
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:52]: There are no tabs -- suggest delete "in the following tabs"

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 3]
```diff
@@ -40,52 +40,223 @@ run by the Kubernetes community (under the aegis of the CNCF).
...
+  - skip excludes the SIG labels listed as optional in
+    [Reference Architecture-2 (RA-2) Chapter 6](../../../ref_arch/kubernetes/chapters/chapter06.md)
 
-| software                | version     |
-|-------------------------|-------------|
-| Functest                | v1.21       |
-| Kubernetes              | v1.21       |
-| xrally-kubernetes       | 1.1.1.dev12 |
+The Reference Conformance suites must be stable and can be executed on real
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:56]: suggest delete "can be" -- only if the tests are to be executed on 'deployments" would the next sentence about "skipping" be releant

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 4]
```diff
@@ -40,52 +40,223 @@ run by the Kubernetes community (under the aegis of the CNCF).
...
+deployments. Then all the following labels are defacto skipped in
+[End-to-End Testing](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md):
+  - alpha
+  - Disruptive
+  - Flaky
+
+It's worth mentioning that no alpha or Flaky test can be included in
+Conformance by rules.
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:64]: suggest replace "by" --> "as per the"

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 5]
```diff
@@ -40,52 +40,223 @@ run by the Kubernetes community (under the aegis of the CNCF).
...
+
+#### Conformance
+
+It must be noted that the default
+[K8s Conformance](https://github.com/cncf/k8s-conformance) is disruptive and
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:69]:
```suggestion
[K8s Conformance](https://github.com/cncf/k8s-conformance) testing is disruptive and
```

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 6]
```diff
@@ -40,52 +40,223 @@ run by the Kubernetes community (under the aegis of the CNCF).
...
+CNTT RC2 rather picks [non-disruptive-conformance](https://sonobuoy.io/docs/master/e2eplugin/)
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:70]:
```suggestion
thus CNTT RC2 uses the [non-disruptive-conformance](https://sonobuoy.io/docs/master/e2eplugin/) testing
```

**Comment by collivier** [comment]
thank you @pgoyal01 . They should be all taken into account.

**Review by rgstori — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**